src/dtemplate.d

```diff
@@ -1774,11 +1774,28 @@ extern (C++) final class TemplateDeclaration : ScopeDsymbol
                                 Expression e;
                                 Type t;
                                 Dsymbol s;
-                                taa.index.resolve(instLoc, sc, &e, &t, &s);
+
+                                uint errors = global.startGagging();
+                                Scope *sco;
+
+                                /* The parameter isn't part of the template
```

Comment:
This is hard enough to understand, so please add a link to the bugzilla issue for when the reader needs clarification.